core/src/test/java/io/questdb/test/PropServerConfigurationTest.java

```diff
@@ -287,10 +287,10 @@ public void testAllDefaults() throws Exception {
 
         TestUtils.assertEquals(new File(root, "db").getAbsolutePath(), configuration.getCairoConfiguration().getRoot());
         TestUtils.assertEquals(new File(root, "conf").getAbsolutePath(), configuration.getCairoConfiguration().getConfRoot());
-        TestUtils.assertEquals(new File(root, "snapshot").getAbsolutePath(), configuration.getCairoConfiguration().getSnapshotRoot());
+        TestUtils.assertEquals(new File(root, TableUtils.CHECKPOINT_DIRECTORY).getAbsolutePath(), configuration.getCairoConfiguration().getCheckpointRoot());
```

Comment:
nit: would be nice to keep an assertion for the legacy dir.

Reply:
done https://github.com/questdb/questdb/pull/4821/commits/f3ee3c9b634661a99ae231e16470ca77c51a3d65